### ISSUE TITLE
Adjust color shade of red letter text 

### ISSUE THREAD

**Issue by timbze** [AndBible/and-bible#1836]
https://support.andbible.org/scp/tickets.php?id=457

> For red-letter text could a settings slider be added to alter its intensity (shade of red). The current shade of red is difficult to distinguish from my choice of black text.

**Comment by tuomas2** [comment]
We could fine-tune the current color or make a addon hack with another color, that user can install if he wants.
But I don't think this is worth slider really. Though I am not also really against implementing one if someone wants...

**Comment by agrogers** [comment]
I will try to do this one at some point.

**Comment by timbze** [comment]
Would it be an issue to just let users pick whatever color they want for "Letters of Christ"?

Someone would like a more blu-ish color:
https://support.andbible.org/scp/tickets.php?id=1819

**Comment by tuomas2** [comment]
Perhaps. If this is a popular demand, we could also quickly make a custom add-on module to change it to another fixed color like blue.

**Comment by agrogers** [comment]
That would be a good option. But I think the original suggestion was to
include it in the 'colors' options area which i think makes sense to me. I
suspect many people who would want the option would not expect it in the
addon area.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
As a first fix I would rather change the default color to something (still reddish preferrably) that is pleasant to most eyes if it is not yet so.

**Comment by Noitarud** [comment]
See response I got for #2778. But the necessary colour will be paler in night mode and deep for day mode.

One hint I can supply for "perfect" colouring is [luminance](https://en.wikipedia.org/w/index.php?title=Hue&oldid=1104119504#24_hues_of_HSL/HSV), the page says red gives 30%, green 59%, blue 11% of total. It would be good to ensure all elements on the program are a similar sum, if you have red/maroon on black, the white should be dimmer else the red seems too dim.

**Comment by tuomas2** [comment]
night color can be set to be completely different than daylight color

**Comment by githubisviolatingthegpl** [comment]
I'd like to second this request. 

The deeply saturated red text is hard to read in dark mode.

**Comment by Noitarud** [comment]
I really liked the shade used in Go Bible (Java phone), #C00000, for day mode.

**Comment by seesmof** [comment]
Guys, please confirm if the setting of Red Letter color is in [/app/bibleview-js/src/components/OSIS/Q.vue](https://github.com/AndBible/and-bible/blob/349eefe3569bc693b28f4f8bc61a78857b4b2e17/app/bibleview-js/src/components/OSIS/Q.vue#L45) in this code:

```js
44 <style lang="scss">
45 $redLetters: rgb(215, 13, 13);
46 .redLetters {
```

_Digits at the beginning are the line numbers_

If it is, i could change the RGB to rgb(115, 0, 0) that is a [deeper red](https://coolors.co/contrast-checker/730000-ffffff) and contrasts well on white screen.

**Comment by tuomas2** [comment]
> Guys, please confirm if the setting of Red Letter color is in ....

Yes. You could adjust that css by overriding .redLetters class in an custom css add-on.
